### PR TITLE
Improve image display

### LINKED ISSUE #20: Improve display for images
Image display can have these improvements .

- swipe functionality to move between adjacent images
- slideshow feature 
- smoother transition
- uniform size

### PR DESCRIPTION
**Pull request by parth-07**
Improved image display gui using [blueimp/gallery](https://github.com/blueimp/Gallery)

**Features**
- Swipe events added for changing images 
- More smooth and uniform transitions , uniform as in image size remains uniform
- Slideshow can be enabled using gui controls or space bar
- gui controls for directly skipping more than 1 images

**Screenshots** 
Screenshot from 2020-12-13 00-42-56
Screenshot_20201213-004635_Chrome

Closes #20

### CODE REVIEW

File: src/user/views/displayImage.mustache [hunk 1]
```diff
@@ -61,85 +72,74 @@
...
-                setImages(currentFileIndex);
+        let galleryItems= [];
+        for(file of files)
+        {
+            let temp = {};
+            temp.title = baseName(file);
+            temp.href = '/public' + file;
+            galleryItems.push(temp);
+        }
+        var gallery = blueimp.Gallery(
```
Comment (manorit2001) [inline src/user/views/displayImage.mustache:126]: - Image name is already being shown in the title bar and having image name shown inside the image also feels kinda repetitive
- Same is the case with arrows as the image thumbnails look a lot better and we don't need arrows, the arrows also feels repetitive and unrequired

If there is a `blueimp` settings, then I would propose these changes to be incorporated before merging this branch to master

**Review by manorit2001 — CHANGES_REQUESTED** [review]
Please look into the changes proposed

**Comment by parth-07** [comment]
Yeah , I agree with all your suggested changes . 
I have removed image name from inside the image as well as next and previous controls inside the image .

Updated screenshots :

Screenshot from 2020-12-13 13-23-05

**Review by manorit2001 — APPROVED** [review]
Looks good

**Pull request merged**